package.json

```diff
@@ -32,7 +32,8 @@
     "lint": "eslint \"**/*.{js,ts,tsx}\"",
     "release": "release-it",
     "clean": "del-cli android/build",
-    "bootstrap": "yarn install"
+    "bootstrap": "yarn install",
+    "prepare": "bob build"
```

Comment:
For the sake of documenting it here, can you mention why this was necessary?

Reply:
Whenever react native module types or index.ts changes, module needs to be rebuild using `bob build` to generate proper type and js files. 

Impact on windows development, need to be discussed